src/components/RequestList/index.tsx

```diff
@@ -206,6 +216,24 @@ const RequestList = () => {
                 {intl.formatMessage(messages.sortModified)}
               </option>
             </select>
+            <Tooltip content={intl.formatMessage(messages.sortDirection)}>
+              <Button
+                buttonType={'ghost'}
+                className="z-40 mr-2 rounded-l-none"
+                buttonSize={'md'}
+                onClick={() =>
+                  setCurrentSortDirection(
+                    currentSortDirection === 'asc' ? 'desc' : 'asc'
+                  )
+                }
+              >
+                {currentSortDirection === 'asc' ? (
+                  <ArrowUpIcon className={'h-3'} />
+                ) : (
+                  <ArrowDownIcon className={'h-3'} />
+                )}
+              </Button>
```

Comment:
Please edit the `{'foobar'}` to `"foobar"` to remain consistent with the rest of the codebase